assets/javascripts/kitten/components/cards/team-card/phone-icon.js

```diff
@@ -32,7 +34,9 @@ class TeamCardPhoneIconBase extends Component {
           </ButtonIcon>
         )}
 
-        {!viewportIsXSOrLess && <TeamCardButtonWithTooltip {...this.props} />}
+        {!viewportIsXSOrLess && (
+          <TeamCardButtonWithTooltip color={tooltipColor} {...this.props} />
```

Comment:
Parfait, ça me semble mieux ! J'intervertirai juste la valeur de `color` par `this.props` pour éviter qu'elle ne soit écrasée.
```suggestion
          <TeamCardButtonWithTooltip {...this.props} color={tooltipColor} />
```